app/tests/src/address-manager.js

```diff
@@ -1,154 +0,0 @@
-import assert from 'assert'
```

Comment:
doesn't this remove test coverage for Byron address, which are still used to migrate the funds from Byron-era wallets?

Reply:
this code indeed seems to test dead code, but I'd rather introduce fixtures to cover Byron-era addresses so they are not left out - arguably those are the easiest to forget about when doing future modifications

Reply:
You are right, There are no other tests for byron address provider so I ll return these tests. Thnx :+1: